.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL = patv9V491TkNGUhyA.0e452983e9bc1b7541a5bcc3029deb4e6882327d620244e2c44b4399f9700c28
```

Comment:
Again @sarthak-stranger you push the `.env` 😢 , Please remove it.
 
 Also please create `.env.example` and just put the variable `NEXT_PUBLIC_API_URL` there and please add in the `README.md` what is the use for this variable.
 
 My two cents are that we should rename `NEXT_PUBLIC_API_URL` to `NEXT_AIRTABLE_API_KEY` as it makes it obvious for the people to understand what it is for.